### ISSUE TITLE
Crash report on 7.0: NFE: Invalid int: "zss_field_content" in onMediaRetryClicked

### ISSUE THREAD

**Issue by hypest** [wordpress-mobile/WordPress-Android#5673]
```
Fatal Exception: java.lang.NumberFormatException: Invalid int: "zss_field_content"
       at java.lang.Integer.invalidInt(Integer.java:138)
       at java.lang.Integer.parse(Integer.java:375)
       at java.lang.Integer.parseInt(Integer.java:366)
       at java.lang.Integer.parseInt(Integer.java:332)
       at java.lang.Integer.valueOf(Integer.java:490)
       at org.wordpress.android.ui.posts.EditPostActivity.onMediaRetryClicked(EditPostActivity.java:2133)
       at org.wordpress.android.editor.EditorFragment.onMediaTapped(EditorFragment.java:1379)
       at org.wordpress.android.editor.JsCallbackReceiver.executeCallback(JsCallbackReceiver.java:160)
       at org.wordpress.android.editor.EditorWebViewAbstract$1.shouldOverrideUrlLoading(EditorWebViewAbstract.java:68)
       at android.webkit.CallbackProxy.uiOverrideUrlLoading(CallbackProxy.java:336)
       at android.webkit.CallbackProxy.handleMessage(CallbackProxy.java:461)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:4898)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1006)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:773)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

**Comment by hypest** [comment]
Probably related as well:
```
Fatal Exception: java.lang.NumberFormatException: Invalid int: "zss_field_content"
       at java.lang.Integer.invalidInt(Integer.java:138)
       at java.lang.Integer.parse(Integer.java:375)
       at java.lang.Integer.parseInt(Integer.java:366)
       at java.lang.Integer.parseInt(Integer.java:332)
       at java.lang.Integer.valueOf(Integer.java:490)
       at org.wordpress.android.ui.posts.EditPostActivity.onMediaUploadCancelClicked(EditPostActivity.java:2140)
       at org.wordpress.android.editor.EditorFragment$20.onClick(EditorFragment.java:1348)
       at android.support.v7.app.AlertController$ButtonHandler.handleMessage(AlertController.java:161)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:4903)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:795)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:562)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

**Comment by mzorz** [comment]
Logs that might be of interest here https://github.com/wordpress-mobile/WordPress-Android/issues/5817#issuecomment-298665184

**Comment by aforcier** [comment]
This looks to be fairly rare, and the underlying cause isn't immediately clear. I'll assign this to myself and try to look into it this week.

**Comment by tonyr59h** [comment]
Closed via #5941.
